### PR TITLE
Add new endpoint for remove all pub addresses

### LINKED ISSUE #20: [TypeScript SDK] Add removeAllAddresses to SDK
Add the new removeAllAddresses call to the TypeScript SDK.

### PR DESCRIPTION
**Pull request by edrotthoff**
this PR adds a new endpoint to the typescript sdk, for removeAllPublicAddresses.
this PR adds new unit tests to verify the new endpoint.

this PR needs a detailed Code review from Andrew before merging.

### CODE REVIEW

File: tests/index.spec.js [hunk 1]
```diff
@@ -392,6 +392,53 @@ describe('Testing generic actions', () => {
...
+  it(`RE Add public addresses`, async () => {
+    const result = await fioSdk.genericAction('addPublicAddresses', {
+      fioAddress: newFioAddress,
+      publicAddresses: [
+        {
+          chain_code: ethChainCode,
+          token_code: ethTokenCode,
+          public_address: 'xxxxxxyyyyyyzzzzzz1',
+        },
+        {
```
Comment (andreyvEze) [inline tests/index.spec.js]: I think remove fio pub address from this array

File: tests/index.spec.js [hunk 2]
```diff
@@ -392,6 +392,53 @@ describe('Testing generic actions', () => {
     expect(result.fee_collected).to.be.a('number')
   })
 
+  it(`RE Add public addresses`, async () => {
+    const result = await fioSdk.genericAction('addPublicAddresses', {
```
Comment (andreyvEze) [inline tests/index.spec.js]: I think we can move this part to the `Remove all public addresses` section (without expect statements), above `const result = await fioSdk.genericAction('removeAllPublicAddresses', {`

File: tests/index.spec.js (hunk 1 above)
Comment (edrotthoff) [reply tests/index.spec.js]: ill do this..

File: tests/index.spec.js (hunk 2 above)
Comment (edrotthoff) [reply tests/index.spec.js]: ill move it...

**Review by ericbutz — APPROVED** [review]

**Pull request merged**